docs_src/microservices/security/Ch-APIGateway.md

```diff
@@ -128,6 +128,40 @@ will be denied when trying to access resources through the API Gateway.
     name = "acl"
     whitelist = "admin,user"
 
+**Configuration of Adding Microservices Routes for API Gateway**
+
+For the current pre-existing Kong routes, they are configured and initialized statically through configuration TOML file specified in `security-proxy-setup` application. This is not sufficient for some other new additional microservices like application services, for example.  Thus, adding new proxy Kong routes are now possibly achieved via environment variable, `ADD_PROXY_ROUTE`, of service `edgex-proxy` in the docker-compose file.  Here is an example:
+
+```yaml
+edgex-proxy:
+      ...
+    environment:
+      ...
+      ADD_PROXY_ROUTE: "myApp.http://my-app:56789"
+      ...
+
+...
+
+myApp:
+   ...
+   container_name: myApp
+   hostname: myApp
+   ...
+  
+```
+
+The value of `ADD_PROXY_ROUTE` takes a comma-separated list of one or more (at least one) paired additional service name and URL for which to create proxy Kong routes.   The paired spec. is given like the following:
+
+**\<ServiceName\>.\<RouteURL\>**
```

Comment:
Is the route case sensitive?

Reply:
s/spec./specification/

Reply:
the service name is case insensitive based on the current implementation: https://github.com/edgexfoundry/edgex-go/blob/master/internal/security/proxy/service.go#L157 
 
the RouteURL is parsed from Golang's package net.url utility and according to this https://github.com/golang/go/issues/3913 closed issue, it is case insensitive as well.

Reply:
Please state as such in the documentation.